seed.js

```diff
@@ -0,0 +1,38 @@
+const data = require('./quotes').quotes;
+const db = require('./database')
```

Comment:
db isn't being used right now. seems like you are creating the `insert` SQL data in this file though, perhaps I would remove this line.